metricbeat/module/windows/service/service_windows.go

```diff
@@ -225,7 +226,12 @@ func getServiceInformation(rawService *EnumServiceStatusProcess, servicesBuffer
 	if ServiceState(rawService.ServiceStatusProcess.DwCurrentState) != ServiceStopped {
 		processUpTime, err := getServiceUptime(rawService.ServiceStatusProcess.DwProcessId)
 		if err != nil {
-			logp.Warn("Uptime for service %v is not available", service.ServiceName)
+			if _, ok := protectedServices[service.ServiceName]; errors.Cause(err) == syscall.ERROR_ACCESS_DENIED && !ok {
+				protectedServices[service.ServiceName] = struct{}{}
+				logp.Warn("Uptime for service %v is not available", service.ServiceName)
```

Comment:
I think this warning could use a bit more explanation about why uptime is not available.